runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabQuantityOverlay.java

```diff
@@ -0,0 +1,89 @@
+package net.runelite.client.plugins.banktags.tabs;
+
+import java.awt.Color;
+import java.awt.Graphics2D;
+import javax.inject.Inject;
+import net.runelite.api.Client;
+import net.runelite.api.Point;
+import net.runelite.api.widgets.WidgetItem;
+import net.runelite.client.game.ItemManager;
+import net.runelite.client.plugins.banktags.BankTagsConfig;
+import net.runelite.client.plugins.banktags.TagManager;
+import net.runelite.client.ui.FontManager;
+import net.runelite.client.ui.overlay.WidgetItemOverlay;
+
+public class TabQuantityOverlay extends WidgetItemOverlay
+{
+	private final Client client;
+
+	@Inject
+	private ItemManager itemManager;
+
+	@Inject
+	private TabInterface tabInterface;
+
+	@Inject
+	private TagManager tagManager;
+
+	@Inject
+	private BankTagsConfig bankTagsConfig;
+
+	@Inject
+	TabQuantityOverlay(Client client, TabInterface tabInterface, TagManager tagManager, BankTagsConfig bankTagsConfig)
+	{
+		this.client = client;
+		this.tabInterface = tabInterface;
+		this.tagManager = tagManager;
+		this.bankTagsConfig = bankTagsConfig;
+		showOnBank();
+	}
+
+	private static String formatNumber(int amount)
```

Comment:
There is a stack formatter class

Reply:
Thanks for all this. I'll fix it all. :)